frontend/src/components/Cup/Cup.scss

```diff
@@ -7,7 +7,6 @@
     box-shadow: 13px 0 64px $yellow;
     border: 5px solid white;
     box-sizing: border-box;
-    margin-bottom: 20px;
```

Comment:
Spacing between elements is best done in parent / container components, and not in the element itself. It should be spacing-agnostic, so to speak :-)